src/actions/toolbox.js

```diff
@@ -37,3 +37,19 @@ export function openWorkerToolbox(url: string) {
     }
   };
 }
+
+export function evaluateInConsole(inputString: string) {
+  return async ({ client, getState }: ThunkArgs) => {
+    const frameId = getSelectedFrameId(getState());
+
+    console.log(inputString);
```

Comment:
oops

Reply:
This one is in order to reassure the person evaluating what they are evaluating before it is being evaluated.